docs/apache-airflow-providers-airbyte/index.rst

```diff
@@ -55,7 +55,7 @@
     :maxdepth: 1
     :caption: Resources
 
-    Example DAGs <https://github.com/apache/airflow/tree/providers-airbyte/|version|/providers/tests/system/airbyte>
+    Example DAGs <https://github.com/apache/airflow/tree/providers-airbyte/|version|/tests/system/providers/airbyte>
```

Comment:
There is no such link though `https://github.com/apache/airflow/tree/providers-airbyte/`

Reply:
This link seems to be working. What am I missing here?
https://github.com/apache/airflow/tree/providers-airbyte/4.0.0/tests/system/providers/airbyte

Reply:
aah my bad